JYU-LB

```diff
@@ -0,0 +1 @@
+Subproject commit c7d07ba1666966f7c1cf5a8202d078240cf43ab4
```

Comment:
A tagged release of JYU-LB should be used.  @kemattil , can you tag a new release.